### ISSUE TITLE
End of timer notification

### ISSUE THREAD

**Issue by bkinnamon** [bkinnamon/pomodoro#4]
When the timer ends there should be a visual notification

**Comment by bkinnamon** [comment]
Maybe "flash" the background a little in accordance with WCAG guidelines

https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure.html
